tests/test_core.py

```diff
@@ -126,5 +126,27 @@ def test_copy_support_files_use_target(self):
         self.files_to_remove.append(local_path)
         self.files_to_remove.append('./something-else')
 
+    def test_custom_axes(self):
+        emp = Emperor(self.ord_res, self.mf)
+        obs = emp.make_emperor(custom_axes=['DOB'])
+
+        with open('test.txt', 'w') as f:
+            f.write(obs)
+
+        try:
+            self.assertItemsEqual(tcs.HTML_STRING_CUSTOM_AXES.split('\n'),
+                                  obs.split('\n'))
+        except AttributeError:
+            self.assertCountEqual(tcs.HTML_STRING_CUSTOM_AXES.split('\n'),
+                                  obs.split('\n'))
+        self.assertEqual(tcs.HTML_STRING_CUSTOM_AXES, obs)
```

Comment:
The other option is to completely drop PY2 support.  There is no reason to keep that anymore.